src/api/kube.ts

```diff
@@ -32,7 +32,7 @@ export class KubeHelper {
   static initializeKubeConfig(): KubeConfig {
     const kc = new KubeConfig()
     kc.loadFromDefault()
-    cli.info(`Set current context to '${kc.currentContext}'`)
+    cli.info(`› Current kuberntetes context: '${kc.currentContext}'`)
```

Comment:
kuberntetes -> kubernetes . And maybe from capital letter.

Reply:
```suggestion
    cli.info(`› Current Kubernetes context: '${kc.currentContext}'`)
```